aries-site/src/examples/templates/forms/FormHeaderExample.js

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Select,
+  Text,
+  TextInput,
+  TextArea,
+} from 'grommet';
+import { CircleAlert } from 'grommet-icons';
+
+const superPower = ['Flying', 'Sky Runner', 'Invisibility'];
+const weakness = ['Fire', 'PB & J', 'Kryptonite'];
+
+const RequiredFormField = props => {
+  const { required, label, ...rest } = props;
+  return (
+    <FormField
+      label={required ? `${label}*` : label}
+      required={required}
+      {...rest}
+    />
+  );
+};
+
+RequiredFormField.propTypes = {
+  required: PropTypes.bool,
+  label: PropTypes.string,
+};
+
+const FormContainer = ({ ...rest }) => {
+  return (
+    <Box background="background-front" border round="small" overflow="hidden">
+      <Box flex pad={{ horizontal: 'medium', vertical: 'medium' }} {...rest} />
+    </Box>
+  );
+};
+
+export const FormHeaderExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    name: 'Enduro',
+    superPower: 'Sky Runner',
+    nemesis: true,
+    email: 'enduro@skyrunner.io.com',
+    weakness: 'PB & J',
+  });
+
+  const onFormChange = value => {
+    setFormValues(value);
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Text size="xxlarge" weight="bold">
+            Form Header
+          </Text>
+        </Header>
+        <Box
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            messages={{
+              required: (
+                <Box
+                  background="background-front"
+                  align="center"
+                  gap="xsmall"
+                  direction="row"
+                >
+                  <CircleAlert size="small" />
+                  <Text size="xsmall">This is a required field.</Text>
+                </Box>
+              ),
+            }}
+            value={formValues}
+            onChange={onFormChange}
+          >
+            <RequiredFormField
+              required
+              error={
+                <Box
+                  background="background-front"
+                  align="center"
+                  gap="xsmall"
+                  direction="row"
+                >
+                  <CircleAlert size="small" />
+                  <Text size="xsmall">Invalid credentials.</Text>
+                </Box>
```

Comment:
Should this be abstracted to an <Error /> or <ErrorMessage /> component similar to your <RequiredField />? It would be more representative of a real world example.

API would simply be:
```
<Error>Invalid credentials.</Error>
```

Reply:
oh right thats a good point I can extract it